include/aws/io/host_resolver.h

```diff
@@ -76,10 +81,21 @@ struct aws_host_listener;
 
 struct aws_host_listener_options;
 
+struct aws_host_resolver_purge_host_options {
+    struct aws_host_resolver *resolver;
+    /* the host to purge the cache for */
+    const struct aws_string *host;
+    /* Callback to invoke when the purge is complete */
+    aws_on_host_purge_complete_fn *on_host_purge_complete_callback;
+    /* user_data will be passed as it is in the callback. */
+    void *on_host_purge_complete_user_data;
```

Comment:
trivial: just rename it to `user_data` or `complete_user_data` as the structure name already have `host_purge`.

Reply:
Thanks, fixed.